src/game/scenes/mainmenu/menu_help.c

```diff
@@ -0,0 +1,106 @@
+#include "game/scenes/mainmenu/menu_help.h"
+
+#include "game/game_state.h"
+#include "game/gui/gui.h"
+#include "game/gui/text/text.h"
+#include "game/protos/scene.h"
+#include "game/utils/nat.h"
+#include "game/utils/settings.h"
+#include "resources/languages.h"
+#include "utils/allocator.h"
+#include "utils/log.h"
+#include "video/video.h"
+
+#define TEXT_SHADOW_GREEN 254
+
+typedef struct {
+    scene *s;
+    text_document *td;
+    uint8_t page;
+    surface sur;
+} help_menu_data;
+
+void help_menu_update(help_menu_data *local) {
+    text_document_free(&local->td);
+    local->td = text_document_create();
+
+    text_margin margin = {10, 0, 0, 0};
+
+    str helpstr;
+    str_from_c(&helpstr, lang_get(local->page));
+
+    text_generate_document(local->td, &helpstr, FONT_BIG, 280, 170, TEXT_BRIGHT_GREEN, TEXT_SHADOW_GREEN,
+                           TEXT_ALIGN_TOP, TEXT_ALIGN_LEFT, margin, 1, 0, 0, 0);
+
+    str_free(&helpstr);
+}
+
+void menu_help_free(component *c) {
+    help_menu_data *local = menu_get_userdata(c);
+    text_document_free(&local->td);
+    surface_free(&local->sur);
+    omf_free(local);
+    menu_set_userdata(c, local);
+}
+
+void menu_help_render(component *c) {
+    help_menu_data *local = menu_get_userdata(c);
+    video_draw(&local->sur, 15, 15);
+
+    text_document_draw(local->td, 15, 20);
+}
```

Comment:
I assume the bottom of the help menu is to come at a later date, when more of the text layout quirks are handled correctly?

Reply:
Yes, the main goal was to get a mostly working help menu for now.